hermes/probe/delete/delete_test.go

```diff
@@ -114,13 +114,14 @@ func genTestTarget(cfg *monitorpb.HermesProbeDef, t *testing.T) *probe.Target {
 	}
 }
 
-// setupFakeSystem sets up the fake storage system through the fake client.
-func setupFakeSystem(ctx context.Context, t *testing.T) stiface.Client {
+// fakeStorageClient sets up the fake storage system through the fake client
+// and returns the fake client.
+func fakeStorageClient(ctx context.Context, t *testing.T) stiface.Client {
 	client := mock.NewFakeClient()
```

Comment:
Rename "mock" to "fakeclient", and "NewFakeClient" to "New".
Then this becomes `client := fakeclient.New()`, which is more idiomatic.

Reply:
I've updated to the latest commit in Alicja's PR for fakes, so it's now:
`fakegcs.NewClient()` which is quite intuitive, I think.